pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -310,7 +316,7 @@ def __recover_child_process_weights(self, best_path, last_path):
             self.lightning_module.load_state_dict(ckpt)
 
     def barrier(self, *args, **kwargs) -> None:
-        if not torch_distrib.is_initialized():
+        if not distributed_available():
```

Comment:
hey, it needs both, available() and initialized()

my bad in the barrier PR #8165, I thought available() is redundant but didn't notice the windows test failing who need this.

Reply:
`distributed_available` has that check

Reply:
I would avoid that because it has an or with `tpu_distributed()` which uses a different library for distributed training and is not compatible with this ddp plugin, but ok not a big deal.